logbook-spring-webflux/src/main/java/org/zalando/logbook/spring/webflux/ClientResponse.java

```diff
@@ -24,7 +22,8 @@ final class ClientResponse implements HttpResponse {
 
     @Override
     public int getStatus() {
-        return response.rawStatusCode();
+        // FIX-ME: May throw NPE
```

Comment:
I am assuming this `FIX-ME` is for a (potential) follow-up and wasn't meant to be addressed in this PR, right?